### PR TITLE
95221 Update school certification upload screen copy - form 10-10d

### PR DESCRIPTION
**Pull request by michaelclement**
## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [X] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

Updates copy on the school certification upload page.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/95221

## Testing done

- Manual

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Desktop | Screenshot 2024-11-08 at 09 51 44|Screenshot 2024-11-08 at 09 47 31|

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/config/form.js [hunk 1]
```diff
@@ -185,6 +185,7 @@ const formConfig = {
       title: 'Signer information',
       pages: {
         page2: {
+          // initialData: mockData.data,
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/config/form.js:188]: Intentionally added. Makes testing easier when I can just comment/uncomment this line as needed.

File: src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js [hunk 2]
```diff
@@ -147,7 +147,10 @@ export function uploadWithInfoComponent(fileList, category, resources) {
           </>
         ),
       },
-      ...fileUploadBlurb,
+      ...fileUploadBlurbCustom(
+        undefined,
+        'If you don’t want to upload your supporting files now, you’ll have the option to upload again at the end of the application.',
+      ),
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js:153]: Switching to use the custom blurb component that supports arbitrary follow-up text.

**Review by stevelong00 — APPROVED** [review]
LGTM!

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**